lib/mime.c

```diff
@@ -1561,6 +1561,14 @@ CURLcode Curl_mime_set_subparts(curl_mimepart *part,
       }
     }
 
+    /* If subparts have alreay been used as a top-level MIMEPOST,
+       they might be not positioned at start. Rewind them now, as
+       future check of parent needing rewind may cause this content to
```

Comment:
maybe "a future check of a parent"?

Reply:
Not very elegant, I admit. Will rephrase.